build.sbt

```diff
@@ -0,0 +1,31 @@
+val slickVersion = "3.1.1"
+
+lazy val mainProject = (project in file(".")).settings(
```

Comment:
does `build.sbt` still support the constructor syntax `Project(...` ? Let's stick with that as a compromise?